staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -199,9 +199,10 @@ type Routes interface {
 }
 
 var (
-	InstanceNotFound = errors.New("instance not found")
-	DiskNotFound     = errors.New("disk is not found")
-	NotImplemented   = errors.New("unimplemented")
```

Comment:
unimplemented => not implemented

Reply:
Is there value in changing text of an existing error? There is some code like 

https://github.com/kubernetes/kubernetes/blob/269183a7a83aa48d3a1da43d57e3f9e94e459bd6/staging/src/k8s.io/legacy-cloud-providers/azure/azure_fakes.go#L874

using this string value.

Reply:
ah -- yuck. Ok, it's unfortunate that there is no consistency, but let's keep it the same.